diff --git a/node_modules/gatsby-source-wordpress-experimental/steps/source-nodes/fetch-nodes/fetch-referenced-media-items.js b/node_modules/gatsby-source-wordpress-experimental/steps/source-nodes/fetch-nodes/fetch-referenced-media-items.js
index ed029bf..08fac6b 100644
--- a/node_modules/gatsby-source-wordpress-experimental/steps/source-nodes/fetch-nodes/fetch-referenced-media-items.js
+++ b/node_modules/gatsby-source-wordpress-experimental/steps/source-nodes/fetch-nodes/fetch-referenced-media-items.js
@@ -419,6 +419,9 @@ async function fetchReferencedMediaItemsAndCreateNodes({
   const state = _store.default.getState();
 
   const queryInfo = state.remoteSchema.nodeQueries.mediaItems;
+
+  if (!queryInfo) return [];
+
   const {
     helpers,
     pluginOptions
diff --git a/node_modules/gatsby-source-wordpress-experimental/utils/fetch-graphql.js b/node_modules/gatsby-source-wordpress-experimental/utils/fetch-graphql.js
index d1262fd..1b5b96d 100644
--- a/node_modules/gatsby-source-wordpress-experimental/utils/fetch-graphql.js
+++ b/node_modules/gatsby-source-wordpress-experimental/utils/fetch-graphql.js
@@ -23,7 +23,12 @@ var _getGatsbyApi = require("./get-gatsby-api");
 
 var _url = _interopRequireDefault(require("url"));
 
-const http = (0, _axiosRateLimit.default)(_axios.default.create(), {
+const http = (0, _axiosRateLimit.default)(_axios.default.create({
+  proxy: {
+    host: 'localhost',
+    port: 8888
+  }
+}), {
   maxRPS: process.env.GATSBY_CONCURRENT_DOWNLOAD || 50
 });
 
@@ -377,9 +382,10 @@ const fetchGraphql = async ({
 
     const responsePath = response.request.path;
 
-    if (path !== responsePath) {
-      throw new Error(`GraphQL request was redirected to ${responsePath}`);
-    }
+    // if (path !== responsePath) {
+    //   console.log(`og path: ${path} vs. ${responsePath}`)
+    //   throw new Error(`GraphQL request was redirected to ${responsePath}`);
+    // }
 
     const contentType = response.headers[`content-type`];
 
