diff --git a/node_modules/gatsby-source-wordpress-experimental/steps/source-nodes/fetch-nodes/fetch-referenced-media-items.js b/node_modules/gatsby-source-wordpress-experimental/steps/source-nodes/fetch-nodes/fetch-referenced-media-items.js
index ed029bf..08fac6b 100644
--- a/node_modules/gatsby-source-wordpress-experimental/steps/source-nodes/fetch-nodes/fetch-referenced-media-items.js
+++ b/node_modules/gatsby-source-wordpress-experimental/steps/source-nodes/fetch-nodes/fetch-referenced-media-items.js
@@ -419,6 +419,9 @@ async function fetchReferencedMediaItemsAndCreateNodes({
   const state = _store.default.getState();
 
   const queryInfo = state.remoteSchema.nodeQueries.mediaItems;
+
+  if (!queryInfo) return [];
+
   const {
     helpers,
     pluginOptions
